api/MediaDevices.json

```diff
@@ -207,9 +223,178 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDevices/getUserMedia",
           "support": {
-            "webview_android": {
+            "chrome": [
+              {
+                "version_added": "52"
+              },
+              {
+                "version_added": "47",
+                "version_removed": "52",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#enable-experimental-web-platform-features",
```

Comment:
This should be `"name": "Experimental Web Platform features",` and `"value_to_set": "Enabled"` to more closely reflect the UI for this